cwolfmap/audio.h

```diff
@@ -3,12 +3,11 @@
 #include <stdint.h>
 #include <string.h>
 
-#include <SDL_audio.h>
-
+#include "byteorder.h"
 #include "common.h"
 
 #define MUSIC_SAMPLE_RATE 44100
-#define MUSIC_AUDIO_FMT AUDIO_S16
+#define MUSIC_AUDIO_FMT AUDIO_S16SYS
```

Comment:
```suggestion
```
this can be removed altogether because it's an SDL symbol

Reply:
ok just warning with music_play.c